gelf/udpwriter.go

```diff
@@ -57,9 +61,9 @@ func numChunks(b []byte) int {
 // New returns a new GELF Writer.  This writer can be used to send the
 // output of the standard Go log functions to a central GELF server by
 // passing it to log.SetOutput()
-func NewWriter(addr string) (*Writer, error) {
+func NewWriter(addr string) (*UDPWriter, error) {
```

Comment:
For consistency I would call this `NewUDPWriter`. That's a breaking change but it's a major update anyway.